src/compose/ComposeBox.js

```diff
@@ -128,7 +129,10 @@ class ComposeBox extends PureComponent {
         <AutoCompleteView text={message} onAutocomplete={this.handleAutoComplete} />
         <View style={[styles.composeBox, { height: totalHeight }]}>
           <View style={componentStyles.bottom}>
-            <ComposeMenu />
+            <ComposeMenu
+              handleMessageChange={this.handleMessageChange}
```

Comment:
The ComposeMenu will be responsible for the actions calling and no need to pass this to it.